tests/features/ecas-register.feature

```diff
@@ -1,11 +1,44 @@
+@api @casMockServer
 Feature: Register through OE Authentication
   In order to be able to have new users
   As an anonymous user of the system
   I need to be able to go to the registration URL
 
+  Background:
+    Given CAS users:
+      | Username    | E-mail                            | Password           | First name | Last name | Department    | Organisation |
+      | jb007       | 007@mi6.eu                        | shaken_not_stirred | James      | Bond      | DIGIT.A.3.001 | external     |
+
   Scenario: Register
     Given I am an anonymous user
-    When I visit "the user registration page"
+    When I visit "cas-mock-server/eim/external/register.cgi"
     # Redirected to the Ecas mockup server.
     Then I should see "Create an account"
     And I should see "This page is part of the EU login mock."
+
+  @cleanup:user
+  Scenario: Register new user with AutoRegister enabled
+    Given the site is configured to register users if not exists
```

Comment:
Behat tests are mean to end-users. Instead of doing this, we should login as a user with CAS settings permissions, do the settings using UI, then logout.